CONTRIBUTING.md

```diff
@@ -19,25 +19,24 @@ you'd like to see documented.
 
 To release a new version of nbvv on PyPI:
 
-Update nbvv/_version.py (set release version, remove 'dev')
-And update js/package.json to have the same version number.
-Then npm install to update js/package-lock.json with current dependencies and version.
-
-git add the updated files and git commit
-`python setup.py sdist upload`
-`python setup.py bdist_wheel upload`
-`git tag -a X.X.X -m 'comment'`
-Update _version.py (add 'dev' and increment minor)
-git add and git commit
-`git push`
-`git push --tags`
+- Update nbvv/_version.py (set release version, remove 'dev')
+- And update js/package.json to have the same version number.
+- Then npm install to update js/package-lock.json with current dependencies and version.
+- git add the updated files and git commit
+- `python setup.py sdist upload`
+- `python setup.py bdist_wheel upload`
```

Comment:
I'm pretty sure these two python setup.py steps are irrelevant and probably should be removed since github-actions now takes care of publishing?
Also I'd double check that the git tag doesn't have to start with `v` (compare with what the gh-actions trigger on)

Reply:
Yeah, the action triggers on any tag starting with v. I think when I've published I just relied on npm version to create the version commit and tag for me (after incrementing the version manually in _version.py). Is that a procedure that makes sense to document here instead, given that it requires a user to check that _version.py and package.json match, and potentially hard reset if not?

Reply:
yes i'd say so-  it's important that package.json and package-lock.json have same version in them, and then the version in nbvv/_version.py is really the one that PyPI cares about for publishing.  Basic idea for simplicity is to just keep them all the same version number.

There's also a python package called bump2version which can actually help simplify all these steps but might be overkill for this pr.